web/src/main/java/com/graphhopper/http/GraphHopperApplication.java

```diff
@@ -43,11 +41,7 @@ public void initialize(Bootstrap<GraphHopperServerConfiguration> bootstrap) {
         bootstrap.addBundle(new RealtimeBundle());
         bootstrap.addCommand(new ImportCommand());
         bootstrap.addCommand(new MatchCommand());
-
-        Map<String, String> resourceToURIMappings = new HashMap<>();
-        resourceToURIMappings.put("/assets/", "/maps/");
-        resourceToURIMappings.put("/META-INF/resources/webjars", "/webjars"); // https://www.webjars.org/documentation#dropwizard
```

Comment:
Apparently this is no longer needed with the default assets bundle. At least maps/pt/ and maps/isochrone/ are working without it.